soda/core/soda/scan.py

```diff
@@ -495,7 +495,7 @@ def run_data_source_scan(self):
                     )
 
     def __checks_to_text(self, checks: list[Check]):
-        return "/n".join([str(check) for check in checks])
```

Comment:
How about `os.linesep` ?

Reply:
I don't think so, that note is in reference to its use as a line terminator, not a file terminator. But I'll try to write some tests to see how it behaves.

Reply:
I think we can live with this for now :)